src/legacy/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -92,150 +163,272 @@ export function getUiSettingDefaults() {
+      description: i18n.translate('kbn.advancedSettings.discover.sortDefaultOrderText', {
+        defaultMessage:
+          'Controls the default sort direction for time based index patterns in the Discover app.',
+      }),
       category: ['discover'],
     },
     'doc_table:highlight': {
-      name: 'Highlight results',
+      name: i18n.translate('kbn.advancedSettings.docTableHighlightTitle', {
+        defaultMessage: 'Highlight results',
+      }),
       value: true,
-      description: `Highlight results in Discover and Saved Searches Dashboard.
-        Highlighting makes requests slow when working on big documents.`,
+      description: i18n.translate('kbn.advancedSettings.docTableHighlightText', {
+        defaultMessage:
+          'Highlight results in Discover and Saved Searches Dashboard. ' +
+          'Highlighting makes requests slow when working on big documents.',
+      }),
       category: ['discover'],
     },
     'courier:maxSegmentCount': {
-      name: 'Maximum segment count',
+      name: i18n.translate('kbn.advancedSettings.courier.maxSegmentCountTitle', {
+        defaultMessage: 'Maximum segment count',
+      }),
       value: 30,
-      description: `Requests in discover are split into segments to prevent massive requests from being sent to
-        elasticsearch. This setting attempts to prevent the list of segments from getting too long, which might
-        cause requests to take much longer to process.`,
+      description: i18n.translate('kbn.advancedSettings.courier.maxSegmentCountText', {
+        defaultMessage:
+          'Requests in discover are split into segments to prevent massive requests from being sent to elasticsearch. ' +
+          'This setting attempts to prevent the list of segments from getting too long, ' +
+          'which might cause requests to take much longer to process.',
+      }),
       category: ['search'],
     },
     'courier:ignoreFilterIfFieldNotInIndex': {
-      name: 'Ignore filter(s)',
+      name: i18n.translate('kbn.advancedSettings.courier.ignoreFilterTitle', {
+        defaultMessage: 'Ignore filter(s)',
+      }),
       value: false,
-      description: `This configuration enhances support for dashboards containing visualizations accessing dissimilar indexes.
-        When set to false, all filters are applied to all visualizations.
-        When set to true, filter(s) will be ignored for a visualization
-        when the visualization's index does not contain the filtering field.`,
+      description: i18n.translate('kbn.advancedSettings.courier.ignoreFilterText', {
+        defaultMessage:
+          'This configuration enhances support for dashboards containing visualizations accessing dissimilar indexes. ' +
+          'When set to false, all filters are applied to all visualizations. ' +
+          'When set to true, filter(s) will be ignored for a visualization ' +
+          `when the visualization's index does not contain the filtering field.`,
```

Comment:
```suggestion
          'when the visualization's index does not contain the filtering field.',
```

Reply:
@Bamieh we don't want to escape `'`, so we use backticks here